sensors/Makefile

```diff
@@ -4,26 +4,49 @@
 # Copyright 2022 Phoenix Systems
 #
 
+LOCAL_PATH := $(call my-dir)
+
 # Sensors Server API Library
 NAME := libsensors
 include $(static-lib.mk)
 
+# FIXME: should be stored and compiled outside 
 # Sensors SPI interface
 NAME := libsensors-spi
 LOCAL_HEADERS := sensors-spi.h
-LOCAL_HEADERS_DIR := nothing  # do not reinstall libsensors.h
+LOCAL_HEADERS_DIR := nothing  # do not reinstall libsensors/*.h
 LOCAL_SRCS := libsensors-spi.c
 DEPS := libzynq7000-gpio-msg libspi-msg
 include $(static-lib.mk)
 
-# Sensors Server
-SENSORS_ALL := imu/lsm9dsxx imu/mpu6000 mag/lsm9dsxx mag/lis2mdl baro/lps25xx baro/ms5611 gps/pa6h gps/ubx
-SENSORS_SIM := imu/imu_sim.c mag/mag_sim.c baro/baro_sim.c gps/gps_sim.c gps/gps_sim.c
-SENSORS_LOCAL := $(SENSORS_SIM) $(addsuffix .c, $(filter $(SENSORS_ALL), $(DEVICE_SENSORS)))
+
+# Simulated sesnsors common library
+NAME := libsimsensors_common
+SRCS := $(wildcard $(LOCAL_PATH)simsensor_common/*.c)
+LOCAL_HEADERS_DIR := nothing  # do not reinstall libsensors/*.h
+DEP_LIBS := libsensors
```

Comment:
`DEP_LIBS` for `static-lib.mk` is invalid and does nothing, there was a typo in safety guard (https://github.com/phoenix-rtos/phoenix-rtos-build/pull/229), but still - you should have found out that it doesn't work :P

This error is common across the code.

Reply:
Well... yeah that`s on my, my mistake. Corrected and tested for correct building of all sensors